src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java

```diff
@@ -84,7 +74,7 @@ public void setUp() throws Exception {
 
         BlobServiceClient blobServiceClient = blobServiceClientBuilder.buildClient();
 
-        String inputContainerName = config.getString("test-storage-container-name");
+        String inputContainerName = STORAGE_CONTAINER_NAME;
```

Comment:
do we need this `inputContainerName` variable  can directly use `STORAGE_CONTAINER_NAME` ?

Reply:
I think this is done for readability of the code.